testassets/ReverseProxy.Code/Startup.cs

```diff
@@ -112,6 +121,11 @@ public void Configure(IApplicationBuilder app)
                 // Custom endpoint selection
```

Comment:
```suggestion
                // Custom endpoint and cluster selection
```

Using `ReassignProxyRequest` makes more sense in the context of the `ab-testing.md` sample.
Perhaps add a comment on when one would want to do this / link to the a/b doc?